CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ##### Additions :tada:
 
 - Updated 3D Tiles Next extensions to match the latest specification. See the [latest specification PR](https://github.com/CesiumGS/3d-tiles/pull/630):
+
```

Comment:
I would omit these newlines, sometimes that can confuse markdown formatters sometimes.